pkg/cmd/logs.go

```diff
@@ -225,14 +225,15 @@ func showLogsFromKubectl(namespace, toMatch, container string) {
 	checkError(err)
 	for _, pod := range pods.Items {
 		if strings.Contains(pod.Name, toMatch) {
-			err := ExecCmd(nil, buildKubectlCommand(namespace, pod.Name, container), false)
+			output, err := ExecCmdReturnOutput("kubectl", kubectl.BuildLogCommandArgs(KUBECONFIG, namespace, pod.Name, container, flags.tail, flags.sinceSeconds)...)
```

Comment:
It is not always the case that the `KUBECONFIG` environment variable will be set. 

The usual flow is to run `gardenctl target ...` and export the output of that command if we want to work with `kubectl` directly but users can also run `gardenctl kubectl ...` without needing to export the `KUBECONFIG` var in their environment

Reply:
The original implementation used it so I kept the same behavior.
```
func buildKubectlCommand(namespace, podName, container string) string {
	var command strings.Builder
	command.WriteString(fmt.Sprintf("kubectl logs --kubeconfig=%s %s%s -n %s ", KUBECONFIG, podName, container, namespace))
```

Reply:
Or you mean that I should handle this case?

Reply:
As we discussed KUBECONFIG is not an environment variable and nothing should be changed in this sense.